doc/sphinx/language/coq-library.rst

```diff
@@ -708,403 +702,7 @@ Tactics
 
 A few tactics defined at the user level are provided in the initial
 state, in module ``Tactics.v``. They are listed at
-https://coq.inria.fr/stdlib/, in paragraph ``Init``, link ``Tactics``.
-
-
-The standard library
```

Comment:
Why do you want to drop this in this PR?  I thought we had consensus to leave the doc in the main repo for now.

Reply:
We have consensus to keep all the documentation about commands and tactics in the main repo (where their OCaml implementation will stay) at least to start with. But this part is about things that are pure Coq/Rocq code, and it makes more sense that their description will live with the code, in the new repo. Otherwise, contributing to the stdlib and modifying the doc couldn't be done in the same PR, it would require two PRs on two different repositories.

Reply:
> it makes more sense that their description will live with the code, in the new repo

I'm fine with that, but the stdlib repo should first provide a way to build a manual of sorts rather than dropping this material first.